src/app/backing/components/AvailableBackingMetric/AvailableBackingMetric.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import * as RadixTooltip from '@radix-ui/react-tooltip'
+import { AvailableBackingMetric } from './AvailableBackingMetric'
+
+const meta: Meta<typeof AvailableBackingMetric> = {
+  title: 'Koto/Backing/Components/AvailableBackingMetric',
+  component: AvailableBackingMetric,
+  parameters: {
+    layout: 'centered',
+  },
+  decorators: [
+    Story => (
+      <RadixTooltip.Provider>
+        <Story />
+      </RadixTooltip.Provider>
+    ),
+  ],
+}
+
+export default meta
+type Story = StoryObj<typeof AvailableBackingMetric>
+
+export const WithAvailableBacking: Story = {
+  args: {
+    availableForBacking: 1000,
+    availableBackingUSD: 50000,
+    onStakeClick: () => console.log('Stake clicked'),
+    onDistributeClick: () => console.log('Distribute clicked'),
```

Comment:
we could add an alert rather than a console log, to make it visible that the buttons have been clicked